.github/workflows/cypress-testing.yml

```
@@ -118,6 +150,219 @@ jobs:
+            -e SERVER_TASKQUEUE_RABBITMQ_URL='amqp://user:password@localhost:5673/' \
+            -e SERVER_AUTH_COOKIE_DOMAIN='localhost:8080' \
+            -e SERVER_AUTH_COOKIE_INSECURE='t' \
+            -e SERVER_GRPC_BROADCAST_ADDRESS='localhost:7077' \
+            -e SERVER_DEFAULT_ENGINE_VERSION='V1' \
+            -e HATCHET_LOG_LEVEL='debug' \
+            ghcr.io/hatchet-dev/hatchet/hatchet-engine:latest \
+            /hatchet/hatchet-engine --config /hatchet/config
+
+          echo "✅ Hatchet Engine started with PID: $(docker ps -q -f name=hatchet_engine)"
+
+          # Start Hatchet Dashboard in background  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)